_includes/manuals/3.9/1.2_release_notes.md

```diff
@@ -0,0 +1,176 @@
+* PATs without expiration date cannot be created through webui ([#36775](https://projects.theforeman.org/issues/36775))
+
+#### Foreman - Web Interface
+* Clear navigation search doesn't clear results ([#36949](https://projects.theforeman.org/issues/36949))
+* Navigation Search doesnt show ansible roles ([#36948](https://projects.theforeman.org/issues/36948))
+* Navigation search menu should be less wide ([#36923](https://projects.theforeman.org/issues/36923))
+* Open bookmarks to the right side ([#36864](https://projects.theforeman.org/issues/36864))
+* Table index page design changes ([#36858](https://projects.theforeman.org/issues/36858))
+* Design the New All Hosts Index page ([#36822](https://projects.theforeman.org/issues/36822))
+* Allow new Table component to be used separately ([#36821](https://projects.theforeman.org/issues/36821))
+* Host details actions kebab can't be closed from plugins ([#36793](https://projects.theforeman.org/issues/36793))
+* Typo in variable name in form for taxonomies ([#36791](https://projects.theforeman.org/issues/36791))
+* Add additional condition for title in PageLayout ([#36783](https://projects.theforeman.org/issues/36783))
+* Register Host page has two titles ([#36734](https://projects.theforeman.org/issues/36734))
+* Documentation links and docs.theforeman.org ([#32848](https://projects.theforeman.org/issues/32848))
+
+#### Installer
+* Make Redis the default cache type ([#36801](https://projects.theforeman.org/issues/36801))
+
+#### Installer - Foreman modules
+* Support Pulpcore 3.39 installations ([#36902](https://projects.theforeman.org/issues/36902))
+* Use HTTP/2 on content proxies to connect to Foreman ([#36854](https://projects.theforeman.org/issues/36854))
+* Allow setting (fc)ct_location ([#36812](https://projects.theforeman.org/issues/36812))
+* Add support for container gateway sqlite timeout tuning ([#36772](https://projects.theforeman.org/issues/36772))
+* Support removal of REX cockpit plugin ([#36090](https://projects.theforeman.org/issues/36090))
+
+#### Installer - foreman-installer script
+* Validate certificates as part of foreman-proxy-certs-generate  ([#36843](https://projects.theforeman.org/issues/36843))
+
+#### Packaging - RPMs
+* Automatically depend on the correct SELinux policy ([#36868](https://projects.theforeman.org/issues/36868))
+
+#### Smart Proxy
+* Add Dependabot Configuration ([#36899](https://projects.theforeman.org/issues/36899))
+* Run foreman-proxy.service with PrivateTmp ([#36890](https://projects.theforeman.org/issues/36890))
+* Raise minimum Ruby version to 2.7 ([#36779](https://projects.theforeman.org/issues/36779))
+
+#### Smart Proxy - Tests
+* Allow clearing the log buffer ([#34163](https://projects.theforeman.org/issues/34163))
+
+*A full list of changes in 3.9.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12]=1765)*
+
+### Release notes for 3.9.1
```

Comment:
We list it from newest to oldest, so first 3.9.1 and then 3.9.0

Reply:
ok switched it back